### PR TITLE
[MRG] Plot topo norm of grad pairs.

### PR DESCRIPTION
**Pull request by jaeilepp**
Taken over from https://github.com/mne-tools/mne-python/pull/2898
Closes https://github.com/mne-tools/mne-python/issues/2884
I'm wondering if we should add a ``Vectorview-norm.lout`` to the repo instead of hacking a custom one from vectorview-all.

### CODE REVIEW

**Comment by agramfort** [comment]
+1 on a Vectorview-grad_norm.lout

**Comment by jaeilepp** [comment]
And should there also be mag channels in that layout? I think @Eric89GXL voted against it in the original PR. Are there any other opinions?

**Comment by agramfort** [comment]
there is already one
https://github.com/mne-tools/mne-python/blob/master/mne/channels/data/layouts/Vectorview-mag.lout

**Comment by jaeilepp** [comment]
Yeah, but my question was more about if we want to show rms and mag channels in the same plot. Maybe we'll just stick to only rms...

**Comment by Eric89GXL** [comment]
@jaeilepp +1 for just the RMS

File: mne/viz/topo.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def _plot_topo(info=None, times=None, show_func=None, layout=None,
                decim=None, vmin=None, vmax=None, ylim=None, colorbar=None,
                border='none', axis_facecolor='k', fig_facecolor='k',
                cmap='RdBu_r', layout_scale=None, title=None, x_label=None,
-               y_label=None, vline=None, font_color='w'):
+               y_label=None, font_color='w'):
```
Comment (jaeilepp) [inline mne/viz/topo.py]: ``vline`` was actually not used here at all.

**Comment by agramfort** [comment]
works like a charm.

2 details that I just noticed. In this plot you see a symmetric colorbar while values are necessarily positive.

figure_3

also on topo if you click on an axis there is no ylabel and units. Here is an example that should say that ii is "Amplitude (fT/cm)"
 
screen shot 2016-02-19 at 22 02 42

**Comment by Eric89GXL** [comment]
@agramfort why is your plot of a topomap instead of a topo?

**Comment by agramfort** [comment]
first is unrelated to this PR. I just saw it when running the example :)

File: mne/viz/evoked.py [hunk 2]
```diff
@@ -519,6 +519,9 @@ def plot_evoked_topo(evoked, layout=None, layout_scale=0.945, color=None,
         The face color to be used for each sensor plot. Defaults to black.
     font_color : str | obj
         The color of text in the colorbar and title. Defaults to white.
+    merge_grads : bool
+        Whether to use RMS value of gradiometer pairs. Only works for neuromag
```
Comment (jasmainak) [inline mne/viz/evoked.py]: Neuromag ;)

File: mne/evoked.py [hunk 3]
```diff
@@ -531,10 +534,11 @@ def plot_topo(self, layout=None, layout_scale=0.945, color=None,
                                 vline=vline, fig_facecolor=fig_facecolor,
                                 fig_background=fig_background,
                                 axis_facecolor=axis_facecolor,
-                                font_color=font_color, show=show)
+                                font_color=font_color, merge_grads=merge_grads,
+                                show=show)
 
     def plot_topomap(self, times="auto", ch_type=None, layout=None, vmin=None,
-                     vmax=None, cmap='RdBu_r', sensors=True, colorbar=True,
+                     vmax=None, cmap=None, sensors=True, colorbar=True,
```
Comment (jaeilepp) [inline mne/evoked.py]: I had to change ``cmap=None`` to make it work. The behavior should be identical to what it was before (apart from colorbar having correct range), so deprecation is not needed.

**Comment by jaeilepp** [comment]
Ready for review. I included the topomap colorbar fix here too.

**Comment by Eric89GXL** [comment]
Do you think it's worth modifying an example to use this?

**Comment by jaeilepp** [comment]
Argh. Topo plotting already consumes quite a bit of time on our docs. What if I just add a comment that this is also possible?

**Comment by Eric89GXL** [comment]
Yeah that's fine.

A bit off-topic, but it just occurred to me that you could likely improve that timing quite a bit by reprogramming the plotting function it to use a single axis, and placing the x/y-axes for each sensor yourself. At least I assume the slowdown occurs due to the multitude of axes. It might not actually be that bad to try. If for some reason people actually want separate axes, we could even have `multiple_axes=False` by default to use the new faster method. WDYT?

**Comment by agramfort** [comment]
LGTM but the ylabel should be "Amplitude (fT/cm)" not just fT/cm. The unit must be in ()

thx @jaeilepp 

+1 @Eric89GXL for the idea of single axis if it still works when clicking on the axes

**Comment by Eric89GXL** [comment]
> LGTM but the ylabel should be "Amplitude (fT/cm)" not just fT/cm. The
unit must be in ()

Or better yet "RMS amplitude (fT/cm)"

> +1 @Eric89GXL for the idea of single axis if it still works when clicking
on the axes

That part would be slightly trickier but disable. @jaeilepp do you have
time or should I try?

**Comment by jaeilepp** [comment]
Ready for review.
> That part would be slightly trickier but disable. @jaeilepp do you have time or should I try?

Go  ahead.

**Comment by agramfort** [comment]
Thx @jaeilepp

**Pull request merged**